cmd/podman/machine/ssh.go

```diff
@@ -35,6 +36,11 @@ func init() {
 		Command: sshCmd,
 		Parent:  machineCmd,
 	})
+	flags := sshCmd.Flags()
+	usernameFlagName := "username"
+	flags.StringVar(&sshOpts.Username, usernameFlagName, "", "Username to use when ssh-ing into the VM.")
+	_ = initCmd.RegisterFlagCompletionFunc(usernameFlagName, completion.AutocompleteDefault)
```

Comment:
This should use `completion.AutocompleteNone`

Reply:
Thanks!